### ISSUE TITLE
item option constants are missing in constant.md file

### ISSUE THREAD

**Issue by AnnieRuru** [HerculesWS/Hercules#2895]
**Describe the bug**
constant.md are missing all the item options constant from db\item_options.conf

**To Reproduce**
```c
prontera,155,185,5	script	djksfhsd	1_F_MARIA,{
	dispbottom DAMAGE_PROPERTY_WATER_USER +"";
	end;
```
it does display 38 no problem but ...

**Expected behavior**
need to show on constant.md

**System specs (please complete the following information):**
 - OS: Windows 10 pro 64-bit
 - Hercules Version: latest (always pull latest git)
 - Packet version: 20190530
 - Mode: Renewal
 - Client type: [2019-05-30aRagexeRE](http://nemo.herc.ws/downloads/2019-05-30aRagexeRE/)
 - Branch: Main

**Plugins used or source modifications**
why do you want to know this ?

**Additional context**

**Comment by Kenpachi2k13** [comment]
> need to show on constant.md

That's impossible because those constants are read from _db/item_options.conf_ on server start.

**Comment by AnnieRuru** [comment]
yes it read from `db/item_options.conf` but didn't output the constant on `doc/constant.md`

https://github.com/HerculesWS/Hercules/blob/stable/doc/constants.md

># Constants

> This document contains all the constants available to the script engine.

yeah but where are all the item option constants ?

**Comment by dastgirp** [comment]
constdb2doc.c needs to be updated for the same, it currently reads item_db.conf only and not item_options.conf

**Comment by MrKeiKun** [comment]
@hemagx this can be closed now
